help/visualize/viewers/sunburst.md

```diff
@@ -2,90 +2,75 @@
+
+The sectors’ size reflects category correlation within the dataset. 
+
+In the example below, you can see that the prominent yellow sector on the outer ring shows people who are `Caucasian`, have a `False` control status and are labeled as `Male`.
 
 ![Sunburst viewer](sunburst-viewer.png)
 
-The rings in a sunburst diagram are divided and sliced
-based on their hierarchical relationship to the parent slice.
-Each sector on the outer rings represents a combination of categories from the inner rings as well.
+## Creation
 
-The size of the sector indicates the relative representation
-of this specific combination of categories within the dataset.
+To create a **Sunburst** viewer, navigate to the **Main Menu**
+and select **Add > Javascript Viewers > Charts > Sunburst**
 
-In the provided picture, the large orange sector on the outer ring represents the
-proportion of individuals who belong to the "Caucasian" race,
-have a "False" control status, and are classified as "Male" in terms of sex.
+Columns with categorical data are automatically added to the sunburst viewer.
 
-## Hierarchical data exploration
+## Data exploration
 
 Use the Sunburst viewer to explore the hierarchical nature of your dataset:
 
 * To **zoom** to a category, click it.
 * To **zoom out**, click the blue circle in the middle.
-* To **reset view**, press Ctrl+Shift+A.
+* To **reset view**, double click on the empty area outside the chart.
 
 ![Sunburst interactive data exploration](sunburst-interactive.gif)
 
-In addition to drilling down into categories, you can select a category by using Ctrl+Click.
-When you select a category in the Sunburst viewer,
-other viewers within the application will respond to the data selection
-and dynamically update their representation.
-
-![Sunburst categories selection](sunburst-categories-selection.gif)
+## Selection 
 
-## Creating a Sunburst viewer
+Sunburst viewer offers several selection options:
 
-To create a **Sunburst** viewer, navigate to the **Main Menu**
-and select **Add > Javascript Viewers > Charts > Sunburst**.
+* To **select** a category, use `Shift` + `Click`.
+* To **invert** selection, use `Ctrl` + `Shift` + `Click`.
 
-When you add a sunburst viewer in Datagrok, the platform selects all columns
-with categorical data.
+Other viewers respond to the data selection and dynamically update the representation.
 
-## Configuring a Sunburst viewer
+![Sunburst categories selection](sunburst-categories-selection.gif)
 
-You can set the used columns,
-and customize some minor visualization options.
-To do that, click the **Gear** icon on top of the viewer and use the **Data**
-and **Misc** info pane
-on the **Context Panel** to manage the viewer’s settings.
+## Molecule rendering
 
-For example, you can:
+A sunburst viewer renders molecular structures on the rings. When you hover over the structure, a label appears above. 
```

Comment:
Propose 'hover over the structure to see the label'

Reply:
Yeah, you are right. That's sounds much better!

Reply:
Great suggestion, Maria.